src/ResourceManager/OperationalInsights/Commands.OperationalInsights.Test/ScenarioTests/QueryTests.cs

```diff
@@ -0,0 +1,37 @@
+﻿using Xunit;
```

Comment:
@csuich2 @alexeldeib it looks like a few `.cs` files that were added are missing a license header -- please add one using the header from any existing file